vng_api_common/notifications/viewsets.py

```diff
@@ -156,13 +155,12 @@ def notify(
             client.create("notificaties", message)
         # any unexpected errors should show up in error-monitoring, so we only
         # catch ClientError exceptions
-        except ClientError as exc:
+        except ClientError:
             logger.warning(
-                json.dumps({
-                    "message": "Could not deliver message to %s" % client.base_url,
-                    "status_code": status_code,
-                    "notification_data": message
-                }), exc_info=True, extra={"notification_msg": message}
+                "Could not deliver message to %s",
```

Comment:
but this won't log the notification to the database right?

Reply:
it will with the changes in open-zaak, I'll tweak that a bit more as well.

In its original form, you have implementation details from open-zaak leaking into a third party library. It's better to keep the third party library clean and pure and stick to basic python logging best practices. With the `extra` keyword, there are sufficient mechanisms to get the log message in the database in Open Zaak.